deploy/deployment.json

```diff
@@ -224,7 +224,7 @@
                             }
                         ],
                         "volumeMounts": [],
-                        "imagePullPolicy": "IfNotPresent"
+                        "imagePullPolicy": "Always"
```

Comment:
It is not needed now. I have added possibility of rewriting it via 'pgo_image_pull_policy:' option in  ./deploy/operator.yaml